src/react_native/core.cljs

```diff
@@ -135,25 +135,27 @@
 (def use-context react/useContext)
 
 (defn use-effect
-  ([effect-fn]
-   (use-effect effect-fn []))
-  ([effect-fn deps]
+  ([handler]
+   (use-effect handler []))
+  ([handler deps]
    (react/useEffect
-    #(let [ret (effect-fn)]
+    #(let [ret (handler)]
        (if (fn? ret) ret js/undefined))
     (bean/->js deps))))
 
-(def use-callback react/useCallback)
+(defn use-callback
+  ([handler]
+   (use-callback handler []))
+  ([handler deps]
+   (react/useCallback handler (bean/->js deps))))
 
-(defn use-effect-once
-  [effect-fn]
-  (use-effect effect-fn))
+(defn use-mount
```

Comment:
do we even need this function? probably it just adds more confusion than anything.

Reply:
good question, at first I removed this function and wrote a guideline on how to use effect, but then I realized it's not so readable and you need some time to find out if it has a function inside or if it has no deps, but actually, it has empty deps, etc, but unmount and mount very verbal, so i would keep them, if someone will use use-effect directly probably that fine ?